### DISCUSSION TITLE
Need help with Ldap Groups

### DISCUSSION THREAD

**Discussion by AleB-Roni** [requarks/wiki#6364] [Help / Questions]
Hello Guys, 

I have a question about the LDAP Strategy. I would like to permit only Users from a specific Group of my Active Directory. I am not the Domain Administrator of this Active Directory. This AD has over 50 thousand Users and I would like to only allow my department to use my wiki. Is there a way to import the group or any other way to make that possible?

![IRPo80sr1X](https://user-images.githubusercontent.com/130990232/232482001-1f823435-9f46-419c-a85d-10c1e9df8d65.png)

![RnOYlZypJx](https://user-images.githubusercontent.com/130990232/232482142-b34fb6e6-5f29-44e4-8de9-a0c63eca1be6.png)

I am able to connect with my AD Account, so everything works till there. 

Thank you in advance

-Roni

**Comment by icsinfo** [comment]
Hello, so the way the wiki LDAP group sync works is the groups you want to sync from AD must also exist on the wiki. So if say you have a group called ITDept in AD that has all your users, create a group on the wiki also called ITDept. **The group name in the wiki must match what is defined in the Group Name Field in AD** for the wiki to add the user to the group. If you have the allow self-registration functionality turned on, any user that is a member of the ITDept group in AD will be granted access to the wiki and be put into the ITDept group in the wiki. 

Groups and their membership are not synced between the two. It is only updated on a user's login. A user that is not a member of any group in the wiki will be granted the same access to your wiki as that of a not logged in user because there won't be any matching group to assign them to. An account profile however will be created on your wiki.

So to accomplish your goal, make it so users without a login can't access any part of the wiki, and create a wiki group with the same name as the AD group that holds your department's users. Make sure to assign that group you create the necessary permissions you want your users to have. Anyone else who logs in will be given a profile unfortunately, but won't see anything because they won't have any groups the wiki can match.

One final thing, if your users are directly added to your AD group, that group search filter will work. If however, they are added by being part of another group, you might want to change the group search filter to (member:1.2.840.113556.1.4.1941:={{dn}}) to enable recursion.

**Reply by AleB-Roni** [reply]
Hi,
Thank you for the quick answer :)
I'll try that today. 
Have a nice day :D
